group_vars/openondemand.yml

```diff
@@ -1,14 +1,14 @@
 ---
 openondemand_auth: basic_pam
-openondemand_jupyter_partition: "compute"
```

Comment:
Note that ALL `compute` nodes actually get configured for OOD jupyter and OOD desktop by the inventory groups at https://github.com/stackhpc/caas-slurm-appliance/blob/main/roles/cluster_infra/defaults/main.yml#L40 (which implements vanilla slurm app `environments/common/layouts/everything`), although OOD itself will now only get configured for those apps on the 1st partition.